core/embed/trezorhal/stm32f4/touch/ft6x36.c

```diff
@@ -21,6 +21,7 @@
 #include TREZOR_BOARD
 
 #include <stdbool.h>
+#include <stdio.h>
```

Comment:
We usually use ifdefs for includes of our own headers but not for the libc headers since it makes little sense and worsens readability. I would rather not to do this.